mod-orders/schemas/composite_po_line.json

```diff
@@ -96,6 +96,15 @@
       "description": "the donor contributing to this purchase order line",
       "type": "string"
     },
+    "donorOrganizationIds": {
+      "description": "donor organization ids associated with this fund",
+      "type": "array",
+      "items": {
+        "description": "UUID of an donor organization record",
+        "$ref": "../../common/schemas/uuid.json"
+      },
+      "index": true
```

Comment:
Just for my understanding, index it provides some documentation that DB index would be applied for this field?

Reply:
I didn't see any documentation, I thought in code review I could get suggestion about creating index for the schema. once learned to create index, it will be removed, because I didn't see any places put index like this

Reply:
Removed